arci/src/traits/laser_scan.rs

```diff
@@ -0,0 +1,21 @@
+use auto_impl::auto_impl;
+
+use crate::error::Error;
+
+#[derive(Clone, Debug, Default)]
+pub struct Scan2D {
+    pub angle_min: f64,
+    pub angle_max: f64,
+    pub angle_increment: f64,
+    pub time_increment: f64,
+    pub scan_time: f64,
+    pub range_min: f64,
+    pub range_max: f64,
+    pub ranges: Vec<f64>,
+    pub intensities: Vec<f64>,
```

Comment:
I feel it's better to add some comments to describe what is it.
Especially I can't understand scan_time means.
(I know that you copied from sensor_msgs...)